packages/core/package.json

```diff
@@ -66,9 +70,10 @@
     "test:update": "yarn test -u",
     "test:coverage": "yarn test -- --coverage",
     "start": "yarn build:esm -- --watch",
-    "build": "yarn build:esm && yarn build:esm:mock-classnames",
+    "build": "yarn build:esm && yarn build:esm:mock-classnames && yarn build:metadata",
```

Comment:
Separate it from build? Since it takes 2min

Reply:
done!